### PR TITLE
Only enable sanity.openjdk tests to hotspot jdk8 s390

### LINKED ISSUE #116: Reduce automated testing for JDK8 Hotspot on s390x
**Summary**

The regular testing for JDK8 Hotspot on s390x is to be reduced to sanity.OpenJDK only.

**Reasoning**

The lack of a JIT on JDK8 Hotspot for s390x results in runtimes for tests being many times longer than on other platforms.

Couple that with finite s390x runtime and minimal demand for this build, and the need for the testing fails to justify the excessive time cost.

This matter was discussed at the recent ACC (link below), and the consensus among attendees was to reduce automated testing for JDK8 Hotspot on s390x to sanity.OpenJDK only.

**PR**

Link to follow.

**Linked Issues**

[TSC issue 218: "Proposal: Scale back Hotspot JDK8 zLinux Effort"](https://github.com/AdoptOpenJDK/TSC/issues/218)
[TSC issue 217: "Bi-weekly Adoptium Community Call - April 5, 2021"](https://github.com/AdoptOpenJDK/TSC/issues/217)

### PR DESCRIPTION
**Pull request by sophia-guo**
Add the ability to specify test list according to vendor or impl

Fix #116 

Signed-off-by: Sophia Guo <sophia.gwf@gmail.com>

### CODE REVIEW

**Review by M-Davies — CHANGES_REQUESTED** [review]
Groovy lint
```
2021-05-03 22:34:10 [INFO]   File:[pipelines/build/common/build_base_file.groovy]
2021-05-03 22:34:16 [ERROR]   Found errors in [npm-groovy-lint] linter!
2021-05-03 22:34:16 [ERROR]   Error code: 1. Command output:
------
GroovyLint: Started CodeNarc Server
/github/workspace/pipelines/build/common/build_base_file.groovy
  220   error    Unexpected input: '{
            def testJobType = "nightly"
            if (releaseType.equals("Weekly") || releaseType.equals("Release")) {
                testJobType = "weekly"
            }
            if (isMap(configuration.test)) {
                if (configuration.test.containsKey(variant)) {
                    //Test is enable for the variant
                    if (configuration.test.get(variant)) {
                        def testObj = isMap(configuration.test.get(variant)
                        if (' @ line 220, column 28.  NglParseError   
```

**Comment by sophia-guo** [comment]
run tests

**Comment by sophia-guo** [comment]
run tests

**Comment by sophia-guo** [comment]
run tests

**Comment by sophia-guo** [comment]
run tests

**Comment by sophia-guo** [comment]
run tests

**Comment by sophia-guo** [comment]
Test build: https://ci.adoptopenjdk.net/job/build-scripts-pr-tester/job/build-test/job/jobs/job/jdk8u/job/jdk8u-linux-s390x-hotspot/3/console

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ NOTE: When the `type` field implies a map, the `String` key of the inner map is
...
+| test                       | ❌    | `String`<br>**OR**<br>`Map<String, List>`<br>**OR**<br>`Map<String, List or Map<String, List>>`   | Case one: tests to run against the binary after the build has completed. A `default` tag indicates that you want to run [whatever the default test nightly/release list is](https://github.com/adoptium/ci-jenkins-pipelines/blob/ab947ce6ab0ecd75ebfb95eb2f75facb83e4dc13/pipelines/build/common/build_base_file.groovy#L66-L88).<br><br>Case two: you can also [specify your own list for that particular platform (not variant)](https://github.com/adoptium/ci-jenkins-pipelines/blob/ab947ce6ab0ecd75ebfb95eb2f75facb83e4dc13/pipelines/jobs/configurations/jdk16_pipeline_config.groovy#L59-L64). <br><br>Case three: or you can even specify the list for that particular platform per variant. The list could be specific one `sanity.openjdk` or `default` (similar to the first case) or a map per nightly or release (similar to case two).
```
Comment (M-Davies) [inline README.md:60]:
```suggestion
| test                       | ❌    | `String`<br>**OR**<br>`Map<String, List>`<br>**OR**<br>`Map<String, Map<String, List>>`   | Case one: Tests to run against the binary after the build has completed. A `default` tag indicates that you want to run [whatever the default test nightly/release list is](https://github.com/adoptium/ci-jenkins-pipelines/blob/ab947ce6ab0ecd75ebfb95eb2f75facb83e4dc13/pipelines/build/common/build_base_file.groovy#L66-L88).<br><br>Case two: You can also [specify your own list for that particular platform (not variant)](https://github.com/adoptium/ci-jenkins-pipelines/blob/ab947ce6ab0ecd75ebfb95eb2f75facb83e4dc13/pipelines/jobs/configurations/jdk16_pipeline_config.groovy#L59-L64). <br><br>Case three: Or you can even specify the list for that particular platform per variant. The list could be specific one `sanity.openjdk` or `default` (similar to the first case) or a map per nightly or release (similar to case two).
```

**Review by M-Davies — CHANGES_REQUESTED** [review]
Couple of styling recommendations

File: README.md (hunk 1 above)
Comment (sophia-guo) [reply README.md]: Updated. Thanks!

**Review by M-Davies — APPROVED** [review]
LGTM. Thanks for doing those changes

**Review by smlambert — APPROVED** [review]

**Pull request merged**